css/style.css

```
@@ -1,3 +1,62 @@
-h1 {
-    color: blue;
+body {
+    margin: 0;
+    font-family: Noto Sans JP;
+    background-color: #fff;
+    color: #333;
+}
+
+.site-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #61472E;
+    color: #fff;
+    padding: 10px 30px;
+}
+
+.logo {
+    width: 300px;
+    height: auto;
+    display: block;
+    margin: 0;
+  }
+
+.header-right .contact-link {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: #fff;
+    text-decoration: none;
+    font-size: 14px;
+}
+
+.mailbox {
+    width: 35px;
+    height: 25px;
```

ここはロゴにならって高さはautoで良きです。SPでも高さを決め打ちしているので、そこは削除で。
```suggestion
    width: 35px;
    height: auto;
```